readme.md

```diff
@@ -176,26 +228,24 @@ User.new
 User.new
 ```
 
+<details>
+  <summary>What can we conclude about the relationship of `def initialize` and `.new`? (Hint: it serves the same purpose as Javascript's constructor function)</summary>
 
-#### Hypothesis Building (2 min)
-
-In the next 2 minutes, write down 3 sentences venturing an educated guess when we might use each of these two methods, `def initialize` and `new`, describing where each would be written.
-
+  > The `initialize` method is run every time `.new` is called.
 
-<details>
-<summary>What can we conclude about the relationship of `def initialize` and `.new`?</summary>
-The `initialize` method is run every time `.new` is called.
 </details>
 
 <details>
-<summary>How is this different from other User methods we've seen?</summary>
-`initialize` and `new` aren't the same word. Going by what else we've seen, we'd expect to see `User.initialize` correspond to `def initialize`. (Under the hood, `.new` is a separate class method that calls the `initialize` instance method.)
+  <summary>How is this different from other User methods we've seen?</summary>
+
+  > `initialize` and `new` aren't the same word. Going by what else we've seen, we'd expect to see `User.initialize` correspond to `def initialize`. (Under the hood, `.new` is a separate class method that calls the `initialize` instance method.)
+
 </details>
 
 
-### You can pass arguments to `initialize`
+### You Can Pass Arguments to `initialize`
 
-`initialize` is a special method in its relationship to `.new`, but otherwise it behaves like any other method. This means you can pass arguments to it:
+`initialize` is a special method in its relationship to `.new`, but otherwise it behaves like any other method. This means you can pass arguments to it...
```

Comment:
could make a reference to the `constructor()` method from javascript to make things easier conceptually, not strictly necessary